src/py42/_internal/clients/storage/storagenode.py

```diff
@@ -0,0 +1,29 @@
+from py42.clients import BaseClient
+import requests
+
+
+class StorageNodeClient(BaseClient):
```

Comment:
can we call this `StoragePreservationDataClient`? All of the other methods in this folder also apply to storage nodes, so this name is a little generic.

Reply:
Changed.